common/lib/protocol-definitions/src/summary.ts

```diff
@@ -21,41 +21,93 @@ export interface ISummaryCommitter {
     date: string;
 }
 
+/**
+ *  Type tag used to distinguish different types of nodes in a {@link ISummaryTree}.
+ */
+// eslint-disable-next-line @typescript-eslint/no-namespace
 export namespace SummaryType {
     export type Tree = 1;
     export type Blob = 2;
     export type Handle = 3;
     export type Attachment = 4;
 
-    export const Tree: Tree = 1 as const;
-    export const Blob: Blob = 2 as const;
-    export const Handle: Handle = 3 as const;
-    export const Attachment: Attachment = 4 as const;
+    /**
+     *  Represents a sub-tree in the summary.
+     */
+     export const Tree: Tree = 1 as const;
+
+     /**
+      * Represents a blob of data that is added to the summary.
+      * Such as the user data that is added to the DDS or metadata added by runtime
+      * such as data store / channel attributes.
+      */
+     export const Blob: Blob = 2 as const;
+
+     /**
+      * Path to a summary tree object from the last successful summary.
+      */
+     export const Handle: Handle = 3 as const;
+
+     /**
+      * Unique identifier to larger blobs uploaded outside of the summary.
+      * Ex. DDS has large images or video that will be uploaded by the BlobManager and
+      * receive an Id that can be used in the summary.
+      */
+     export const Attachment: Attachment = 4 as const;
 }
```

Comment:
If you happen to know the answer, it would be good to document if there was an explicit choice not to declare SummaryType as an enum?

Reply:
@anthony-murphy  do you know why ?

Reply:
@anthony-murphy  Do you know why ?

Reply:
Maybe because of the reason mentioned in this comment here - https://github.com/microsoft/FluidFramework/pull/9548#discussion_r837741346